File: Components/Services/File.vb

Imports DotNetNuke.Common.Utilities

Namespace DNNStuff.SQLViewPro.Services
    Public Class File
        Public Shared Function GetFilenameFromFileId(ByVal fileId As String) As String
            Dim fi As DotNetNuke.Services.FileSystem.IFileInfo = DotNetNuke.Services.FileSystem.FileManager.Instance.GetFile(Convert.ToInt32(UrlUtils.GetParameterValue(fileId)))
            If fi IsNot Nothing Then Return fi.PhysicalPath
            Return ""
        End Function
    End Class
End Namespace

File: Components/Services/File.cs


using System;


using DotNetNuke.Common.Utilities;
	
	
	
	namespace DNNStuff.SQLViewPro.Services
	{
		public class File
		{
			public static string GetFilenameFromFileId(string fileId)
			{
				var fi = DotNetNuke.Services.FileSystem.FileManager.Instance.GetFile(Convert.ToInt32(UrlUtils.GetParameterValue(fileId)));
			if (fi != null)
			{
				return fi.PhysicalPath;
			}
			return "";
		}
	}
}


